test/http-context.test.js

```diff
@@ -0,0 +1,89 @@
+var request = require('supertest');
+var HttpContext = require('../lib/http-context');
+var SharedMethod = require('../lib/shared-method');
+var expect = require('chai').expect;
+
+describe('HttpContext', function () {
+  beforeEach(function () {
+    var test = this;
+  });
+  describe('ctx.args', function () {
+
+    describe('arguments with a defined type (not any)', function () {
+      it('should include a named string arg', givenMethodExpectArg({
+        type: 'string',
+        input: 'foobar',
+        expectedValue: 'foobar'
+      }));
+      it('should coerce integer strings into actual numbers', givenMethodExpectArg({
+        type: 'number',
+        input: '123456',
+        expectedValue: 123456
+      }));
+      it('should coerce float strings into actual numbers', givenMethodExpectArg({
+        type: 'number',
+        input: '0.123456',
+        expectedValue: 0.123456
+      }));
+
+      // NEEDS DISCUSSION: this test fails...
+      // not sure what we should do here - add an option to accept "null" as `null`?
+      // it('should coerce null strings into null', givenMethodExpectArg({
+      //   type: 'string',
+      //   input: 'null',
+      //   expectedValue: null
+      // }));
```

Comment:
Well, since we cannot really tell whether the user wanted to pass a string with the value `"null"`, or a null string, I'd say let's stick to the current behaviour.

You should uncomment the test and fix the expected value to describe the current behaviour. That way we won't accidentally introduce a regression in the future.

Please consider adding `null` tests for other types like `int` and `float` too, so that we have a complete running specification of strong-remoting's behaviour.